test/define-class-test.js

```diff
@@ -160,6 +160,34 @@ QUnit.skip("JavaScript setters work", function(assert) {
 	assert.equal(faves.color, "blue", "Did not change");
 });
 
+QUnit.test("Setters on the define override those on the prototype", function(assert) {
```

Comment:
Maybe we should add a comment saying we're testing this but not documenting it? So if we come back and change this, we know it's ok to change this test.

Reply:
I added a comment.